src/main/web/js/app/capture-search-redirects.js

```diff
@@ -0,0 +1,26 @@
+$(function() {
+  // Get the referrer from the url
+  var url = window.location.href
+  var referrer = getParameterByName("referrer", url)
+  if (referrer != null) {
```

Comment:
Because a falsy value could potentially be `undefined` instead of `null` I've suggest replacing the 
```
if (something != null) {
// do a thing
}
```
... with something like 

```
if (!something) {
return;
}
// do a thing
```